### DISCUSSION TITLE
Is it Neo-Launcher or Omega-Launcher

### DISCUSSION THREAD

**Discussion by archer46** [otakuhqz/Omega#158] [Q&A] [answered]
Hello, getting a bit confused here as I look around GitHub, IzzyOnDroid, and other sites. Some places refer to this project as Omega Launcher, and others Neo-Launcher. I'm guessing here that Omega Launcher was an old name, and is slowly being replaced by the name Neo-Launcher? That being said there doesn't seem to be much of an answer to this anywhere.

**Comment by machiav3lli** [comment, marked as answer]
Yes, the project had the name Omega Launcher and since almost two weeks, we announced the new Neo Applications and this project got renamed to Neo Launcher, there's no releases (not even test releases - thus this will change soon) with the new name yet.
